govcd/edgegateway.go

```diff
@@ -186,6 +272,124 @@ func (eGW *EdgeGateway) RemoveNATPortMapping(natType, externalIP, externalPort s
 
 }
 
+// AddDNATRule creates firewall DNAT rule and return refreshed existing NAT rules array or error
+func (eGW *EdgeGateway) AddDNATRule(ruleDetails NatRule) ([]*types.NatRule, error) {
```

Comment:
Please _extend_ the comments of all new `Add...` functions to make it clear that they support assigning specific network in contrast to the old function (please give the name of the old function as well). This way we will always remember why we have different functions and it will be easy to not forget that there is also the old one.

Reply:
Added